launcher/ui/dialogs/ModUpdateDialog.cpp

```diff
@@ -236,6 +236,10 @@ auto ModUpdateDialog::ensureMetadata() -> bool
         if (skip_rest)
             continue;
 
+        if (candidate->type() == ResourceType::FOLDER) {
+            continue;
+        }
```

Comment:
```suggestion
        if (skip_rest || candidate->type() == ResourceType::FOLDER)
            continue;
```
Just combine the two conditions

Reply:
I don't think that's necessary